index.html

```diff
@@ -90,6 +121,48 @@
 
 	<script>
 
+	$(document).ready(function() {
```

Comment:
done. Fixed indentation in whole index.html file in ddd8195b0a6 be aware that this makes reading the diff from this point impossible in this PR.